src/main/resources/locales/en/system-admin-dashboard.json

```diff
@@ -12,6 +12,11 @@
       "description": "Upload CSV location reference data",
       "link": "manual-reference-data-upload"
     },
+    {
```

Comment:
These should be added to the 'cy' folder too even though there is no Welsh. Same for the other resources

Reply:
added :) thanks.